### DISCUSSION TITLE
Audio clip submissions for detection model improvement?

### DISCUSSION THREAD

**Discussion by tphakala** [birdnet-team/BirdNET-Analyzer#260] [General]
Do you accept audio clip submissions for model improvement? 
For example following dog bark is detected at 99% accuracy as Eurasian eagle-owl, in fact 90% of owl detections are dogs in my samples.

[bubo_bubo_99p_20240203T215153Z.zip](https://github.com/kahst/BirdNET-Analyzer/files/14341072/bubo_bubo_99p_20240203T215153Z.zip)

**Comment by pddpauw** [comment]
Yes, would like to know indeed... I recently had several Grus grus identifications (up to 98% confidence level), but our birder community is sure this should be Anser anser.  
Where could we submit the files for this?

**Comment by tphakala** [comment]
Is there any hope we could help improving prediction patterns by providing incorrectly classified audio files?

**Comment by tphakala** [comment]
I have solved this by training my own classifier to improve dog etc. detection
